### DISCUSSION TITLE
想问下作者的编译环境是怎么样的

### DISCUSSION THREAD

**Discussion by jeffreyxuworld** [hefengbao/jingmo#46] [General]
我这边项目编译会报错，用不同的AS版本试了也不行。想问下作者用的具体AS什么版本、Java用的什么版本？

**Comment by hefengbao** [comment]
Android Studio 用最新版本， Jdk 用自带的

**Comment by hefengbao** [comment]
我在 README.md 中添加了一点说明，你可以拉取代码后看下
